sci-mathematics/pari/pari-2.11.3.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="Computer-aided number theory C library and tools"
+HOMEPAGE="http://pari.math.u-bordeaux.fr/"
```

Comment:
https://pari.math.u-bordeaux.fr/
maybe also for SRC_URI ?

Reply:
Right, this should be done indeed. But the other ebuild already present should be touched too. But that would be a separate PR of course.